pkg/apis/resource/types.go

```diff
@@ -579,6 +671,9 @@ type DeviceConstraint struct {
 	// constraint. If this is not specified, this constraint applies to all
 	// requests in this claim.
 	//
+	// References to subrequests must include the name of both the parent request
+	// and the subrequest using the format <parent request>/<subrequest>.
```

Comment:
Is `<parent request>` without the `<subrequest>` allowed? This statement here is about references to subrequests, which doesn't cover references to requests which contain subrequests.

Reply:
Whatever the answer is, are there validation testcases with this?